### PR TITLE
Upgrade meow, @preconstruct/cli, jest & tty-table

### PR DESCRIPTION
**Pull request by zzarcon**
* Bump meow from `5.0.0` to `6.0.0`

The ultimate goal of this PR is to bump `yargs-parser` to a >= 13.1.2 so it contains a security fix

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -7149,51 +7547,27 @@ yargs-parser@^10.0.0:
...
-  dependencies:
-    camelcase "^5.0.0"
-    decamelize "^1.2.0"
-
-yargs-parser@^15.0.0:
-  version "15.0.0"
-  resolved "https://registry.yarnpkg.com/yargs-parser/-/yargs-parser-15.0.0.tgz#cdd7a97490ec836195f59f3f4dbe5ea9e8f75f08"
-  integrity sha512-xLTUnCMc4JhxrPEPUYD5IBR1mWCK/aT6+RJ/K29JY2y1vD+FhtgKK0AXRWvI262q3QSffAQuTouFIKUuHX89wQ==
+yargs-parser@^18.1.2, yargs-parser@^18.1.3:
+  version "18.1.3"
```
Comment (Andarist) [inline yarn.lock:7551]: is this version satisfying?

**Comment by Andarist** [comment]
Is updating `@manypkg/cli` required as part of this PR? It's only a dev dep and more important is to fix our production dependencies.

**Comment by zzarcon** [comment]
@Andarist thanks for having a look into the PR! 

You are right, changing `@manypkg/cli` should not be required since it's a dev dep. I will revert that now

File: yarn.lock (hunk 1 above)
Comment (zzarcon) [reply yarn.lock:7551]: Yeah! The fix was introduced in `13.1.2`

**Review by Andarist — APPROVED** [review]

**Comment by zzarcon** [comment]
Thanks!!

**Pull request merged**